### ISSUE TITLE
Videos not scanning and nothing in app.log

### ISSUE THREAD

**Issue by dazeoff** [chpoit/utsushis-charm#40]
**Device**
- Switch
  - Native screen recording and 720 capture
- Game language: English
- App version: 1.7

** Problem(s) **
I've attempted on 2 PCs and a a Mac, same issue on all three. Given that, I think there is a strong possibility the problem is me. It launches and runs with no errors, the app.log is empty and it doesn't seem to have seen the video. No frames are created. Here's the output given in the window

> Input video location:  /Users/Admin/Downloads/utsushis-charm-1.7.0/inputs
Frame location:  /Users/Admin/Downloads/utsushis-charm-1.7.0/frames
Autosaving Charms:  True
Skipping frame extraction:  False
Deleting existing frames:  False
Extracting Frames
Total input files to scan: 0
Reduced frames from 0 to 0
Extracting charms from frames
Language pack was found
Your charms were saved to "charms.json" and "charms.encoded.txt".
To use on MHR-Wiki, you'll want the contents of "charms.encoded.txt"
Charm extration complete

I've attached the recording I pulled form the switch sd card, I've also tried using a 720 capture with the same issue though. 

https://user-images.githubusercontent.com/75280988/179432701-416d51af-fcf9-45ed-a0e5-d5e720bbbf25.MP4

**Comment by chpoit** [comment]
When you ran it on a mac, was it from built from source? Have you tried running from source on your PCs? From what I see, your recording seems perfectly fine, I will try to run your clip on my computer to see if it works. 

If it doesn't I will try to figure it out during the week.

**Comment by chpoit** [comment]
I was able to extract your charms from your clip. So I definitely think it's a "you" issue.

I saw that you were working in the admin user directory, the app might not be able to access the contents of your "inputs" directory.

In any case, here's what I extracted from your clip:
[charms.encoded.txt](https://github.com/chpoit/utsushis-charm/files/9135623/charms.encoded.txt)

I wont do this every time, but if you want to upload all your clips to a google drive or something so I can extract them for you while you figure it out, feel free to go ahead and do so.
